docs/development/index.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+***********************
+Contributing to Astropy
+***********************
+
+So you are interested in contributing to the Astropy Project?  Excellent! We
+love contributions! Astropy is open source, built on open source, and we'd
+love to have in our community.
+
+For the time being, please see our `CONTRIBUTING.md
+<https://github.com/astropy/astropy/blob/main/CONTRIBUTING.md>`_ document
+for an overview of ways you can contribute to Astropy.  The rest of this
+document is focused specifically on documentation for contributors and
+maintainers on how to work the Astropy repository for submitting
+documentation and code changes.
+
+For the guiding vision of this process and the project
```

Comment:
I don't think this is needed in this context.

Reply:
I don't know.  It used to be on the main index page of the docs.  I could maybe move it back there--I kind of agree it doesn't belong here.